src/pages/ProductList/common/ListSidebar.js

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faWonSign,
+  faLocationDot,
+  faStar,
+  faMap,
+  faFilter,
+  faList,
+} from '@fortawesome/free-solid-svg-icons';
+import FlexibleBtn from '../components/FlexibleBtn';
+
+const ListSidebar = ({ hotelData, setView }) => {
+  const [showFilter, setShowFilter] = useState(false);
+  const [price, setPrice] = useState([]);
+  const [wonValue, setWonValue] = useState(0);
+  const [locationValue, setLocationValue] = useState(0);
+  const [starValue, setStarValue] = useState(1);
+
+  const icon = [faList, faMap, faFilter];
+
+  useEffect(() => {
+    hotelData.forEach(data => {
+      setPrice(price => price.concat(data.price));
+    });
+  }, [hotelData]);
+
+  const listView = () => {
+    setView(true);
+  };
+
+  const mapView = () => {
+    setView(false);
+  };
+
+  const filterView = () => {
+    setShowFilter(!showFilter);
+  };
+
+  const seekbarController = e => {
+    if (e.target.id === 'won') {
+      setWonValue(e.target.valueAsNumber);
+    } else if (e.target.id === 'location') {
+      setLocationValue(e.target.valueAsNumber);
+    } else if (e.target.id === 'star') {
+      setStarValue(e.target.valueAsNumber);
+    }
```

Comment:
```suggestion
  const seekbarController = e => {
const {id, valueAsNumber} = e.target
    if (id === 'won') {
      setWonValue(valueAsNumber);
    } else if (id === 'location') {
      setLocationValue(valueAsNumber);
    } else if (id === 'star') {
      setStarValue(valueAsNumber);
    }
```
* 이렇게 구조분해할당해서 반복되는 코드 줄이면 좀더 명시적이고 깔끔해지겠네요!

Reply:
꿀팁 감사합니다!!!